tracer/test/test-applications/integrations/Samples.AWS.SimpleNotificationService/AsyncHelpers.cs

```diff
@@ -26,6 +26,9 @@ public static async Task StartSNSTasks(AmazonSimpleNotificationServiceClient sns
                 await PublishMessageAsync(snsClient, topicArn);
 #if AWS_SNS_3_7_3
                 await PublishBatchAsync(snsClient, topicArn);
+#else
+                // do a second publish if batch is not available to keep a constant number of spans and make assertions simpler
+                await PublishMessageAsync(snsClient, topicArn);
```

Comment:
I guess this is okay but I'm unsure how I really feel about this as it is a bit misleading. The span  count will be consistent, but not the operations between them.

Reply:
hmm, yeah, but the operations are reflected in the snapshots, which are different anyway. Having this operation really makes the code simpler imho 😶